channeldb/invoice_test.go

```diff
@@ -578,6 +578,67 @@ func TestDuplicateSettleInvoice(t *testing.T) {
 	}
 }
 
+// TestFetchAllInvoices tests that FetchAllInvoices works as expected.
+func TestFetchAllInvoices(t *testing.T) {
+	t.Parallel()
+
+	db, cleanUp, err := makeTestDB()
+	defer cleanUp()
+	if err != nil {
+		t.Fatalf("unable to make test db: %v", err)
+	}
+
+	const numInvoices = 100
```

Comment:
Why so many? One or two in every state seems enough to me

Reply:
Yeah, with 4 states and mod, it's plenty more than needed. Changed.